discretisedfield/field.py

```diff
@@ -2055,7 +2072,7 @@ def grad(self):
             msg = f"Cannot compute gradient for nvdim={self.nvdim} field."
             raise ValueError(msg)
 
-        return self.derivative("x") << self.derivative("y") << self.derivative("z")
+        return self.diff("x") << self.diff("y") << self.diff("z")
 
     @property
     def div(self):
```

Comment:
The double backslashes in the docstring have to be changed to single backslashes (because it is a raw string). Probably also in the other docstrings.